src/cdk-experimental/menu/menu-item.ts

```diff
@@ -61,10 +106,17 @@ export class CdkMenuItem {
     return this.checked;
   }
 
-  /** Whether the menu item opens a menu */
-  hasSubmenu() {
-    return !!this._menuPanel;
+  /**
+   * Toggle the checked state of the menuitemradio or menuitemcheckbox component
+   */
+  private _toggleCheckedState(selected: CdkMenuItem) {
+    if (this.role === 'menuitemradio') {
+      this.checked = selected === this;
+    } else if (this.role === 'menuitemcheckbox' && selected === this) {
```

Comment:
I wonder if it would be a good idea to save 'menuitemradio' and 'menuitemcheckbox' as constants somewhere, so any typos become evident at compile time (instead of runtime).

Reply:
These would actually be enforced by the type system already because they're string literal types